### PR TITLE
remove scrutinizer configs and references

### PR DESCRIPTION
**Pull request by punkrokk**
* Remove `scrutinizer.yml`
* Remove badge on Readme.md
* Add Changelog entry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+[![Tests Build Status](https://travis-ci.org/StackStorm/st2.svg?branch=master)](https://travis-ci.org/StackStorm/st2) [![Packages Build Status](https://circleci.com/gh/StackStorm/st2/tree/master.svg?style=shield)](https://circleci.com/gh/StackStorm/st2) [![Codecov](https://codecov.io/github/StackStorm/st2/badge.svg?branch=master&service=github)](https://codecov.io/github/StackStorm/st2?branch=master) ![Python 2.7](https://img.shields.io/badge/python-2.7-blue.svg) [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1833/badge)](https://bestpractices.coreinfrastructure.org/projects/1833) [![Join our community Slack](https://stackstorm-community.herokuapp.com/badge.svg)](https://stackstorm.com/community-signup) [![Forum](https://img.shields.io/discourse/https/forum.stackstorm.com/posts.svg)](https://forum.stackstorm.com/)
```
Comment (armab) [inline README.md]: Not related to Scrutinizer.

Just noticed py `2.7` badge is present in README.md https://img.shields.io/badge/python-2.7-blue.svg, while py `3.6` is missing. As it's very minor, - could be added in this or another PR, if you wish.

Comment (punkrokk) [reply README.md]: Omg. Sorry. I pulled too much out. I’ll fix it.

Comment (armab) [reply README.md]: The diff looks good.

See https://github.com/StackStorm/st2/pull/4849/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8

Just talking for a py 3.6 badge which we missed to add before.

Comment (punkrokk) [reply README.md]: I updated the python badge to add 3.6

Comment (armab) [reply README.md]: :+1:

**Pull request merged**